### ISSUE TITLE
Content appearing twice per page

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/genesis-SB#3]
I don't know when this started but on every page the content appears twice, the content appears twice.

**Comment by bobbingwide** [comment]
In the functions.php file I'd commented out a line of code, adding a @TODO.
```
// Start the engine	- @TODO Is this necessary?
//include_once( get_template_directory() . '/lib/init.php' );
```

It turns out that this line is necessary because...
- Genesis source files contain code that gets executed when the file is loaded
- lib/init.php invokes 'genesis_init' which calls 
- genesis_load_framework() wich loads lib/structure/post.php
- which performs add_action( 'genesis_entry_content', 'genesis_do_post_content' );

If lib/init.php is not loaded early then any attempts to alter the standard genesis hooks are ineffective, since they've not yet been defined.  

### TL:DR
Yes, you need to start the Genesis engine before making modifications. 
Solution: Uncomment the code, change the @TODO comment.
